crates/sui/src/client_commands.rs

```diff
@@ -1329,20 +1341,28 @@ impl Display for SuiClientCommandResult {
                 )?;
                 writeln!(writer, "{}", ["-"; 165].join(""))?;
                 for oref in object_refs {
+                    // TODO: change this to match behaviors of others (jianhlu)
```

Comment:
@666lcz What is the proper approach to handle the option addition to the `SuiObjectInfo` struct here?

Reply:
Synced offline.

`get_objects_owned_by_address` should reuse the `SuiObjectResponseOptions` (as opposed to adding `SuiObjectInfoOptions`)and return a `Page<SuiObjectResponse, ObjectID>`  (as opposed to `Page<SuiObjectInfo, ObjectID>` ). we should remove `SuiObjectInfo` since it's just a subset of SuiObjectResponse

Reply:
> What is the proper approach to handle the option addition to the SuiObjectInfo struct here?

You can return `anyhow!("Expect the owner type not to be empty")` since we have set SuiObjectResponseOptions.show_owner to be true